server/neptune/temporalworker/server.go

```diff
@@ -111,7 +112,12 @@ func NewServer(config *config.Config) (*Server, error) {
 		Logger:      config.CtxLogger,
 	}
 
-	deployActivities, err := workflows.NewDeployActivities(config.DeploymentConfig)
+	session, err := aws.NewSession()
+	if err != nil {
+		return nil, errors.Wrap(err, "initializing new aws session")
+	}
+
```

Comment:
Can we just keep this initialized down in the activities since nothing else uses this right now?